### PR TITLE
Fix FPs found in testing env

### PR DESCRIPTION
**Pull request by phantinuss**
### Summary of the Pull Request
Fix FPs found in testing env

### SigmaHQ Rule Creation Conventions

- If your PR adds new rules, please consider following and applying these [conventions](https://github.com/SigmaHQ/sigma-specification/blob/main/sigmahq/sigmahq_conventions.md)

### CODE REVIEW

File: rules/windows/process_access/proc_access_win_susp_proc_access_lsass_susp_source.yml [hunk 1]
```diff
@@ -116,6 +116,12 @@ detection:
         SourceImage|endswith: '\updater.exe'
         TargetImage|endswith: '\winlogon.exe'
         GrantedAccess: '0x1fffff'
+    filter_optional_adobe_arm_helper:
+        SourceImage|startswith:  # example path: 'C:\Program Files (x86)\Common Files\Adobe\ARM\1.0\Temp\2092867405\AdobeARMHelper.exe'
+            - 'C:\Program Files\Common Files\Adobe\ARM\'
+            - 'C:\Program Files (x86)\Common Files\Adobe\ARM\'
+        SourceImage|endswith: 'AdobeARMHelper.exe'
```
Comment (nasbench) [inline rules/windows/process_access/proc_access_win_susp_proc_access_lsass_susp_source.yml:123]:
```suggestion
        SourceImage|endswith: '\AdobeARMHelper.exe'
```

File: rules/windows/process_access/proc_access_win_susp_proc_access_lsass_susp_source.yml [hunk 2]
```diff
@@ -116,6 +116,12 @@ detection:
         SourceImage|endswith: '\updater.exe'
         TargetImage|endswith: '\winlogon.exe'
         GrantedAccess: '0x1fffff'
+    filter_optional_adobe_arm_helper:
+        SourceImage|startswith:  # example path: 'C:\Program Files (x86)\Common Files\Adobe\ARM\1.0\Temp\2092867405\AdobeARMHelper.exe'
```
Comment (nasbench) [inline rules/windows/process_access/proc_access_win_susp_proc_access_lsass_susp_source.yml:120]:
```suggestion
        SourceImage|startswith:  # Example path: 'C:\Program Files (x86)\Common Files\Adobe\ARM\1.0\Temp\2092867405\AdobeARMHelper.exe'
```

**Review by nasbench — APPROVED** [review]

File: rules/windows/process_creation/proc_creation_win_hktl_certify.yml [hunk 3]
```diff
@@ -27,11 +28,11 @@ detection:
             - '.exe download '
     selection_cli_options:
         CommandLine|contains:
-            - 'vulnerable '
-            - 'template'
-            - 'altname'
-            - 'domain'
-            - 'path'
+            - ' /vulnerable '
```
Comment (nasbench) [inline rules/windows/process_creation/proc_creation_win_hktl_certify.yml:31]:
```suggestion
            - ' /vulnerable'
```

Comment (nasbench) [reply rules/windows/process_creation/proc_creation_win_hktl_certify.yml:31]: Reducing this space to cover an additional case. Hope this doesn't conflict with the FP that was found? Let me know

Comment (phantinuss) [reply rules/windows/process_creation/proc_creation_win_hktl_certify.yml:31]: no, it's fine.

**Pull request merged**